spec/controllers/admin/user_reports_controller_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe Admin::UserReportsController do
+  describe '#index' do
+    context 'when called by an admin user' do
+      it 'should return a CSV' do
+        user = create(:admin_user)
+        10.times { create(:user) }
```

Comment:
in that case, thoughts on not creating these records? would make tests faster ;)